### PR TITLE
Add output cluster_node_resource_group

### PR DESCRIPTION
**Pull request by gregberns**
This PR outputs the name of the "Nodes" resource group that hosts the VMs.

There's also an update to the docs. Just a note, it looks like the output and maybe inputs are a bit out of date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +127,8 @@ The following sections provide a full list of configuration in- and output varia
 | cluster\_fqdn | The FQDN of the created cluster |
 | cluster\_name | The name of the created cluster |
 | cluster\_resource\_group | Resource group name that contains AKS managed cluster |
+| cluster\_node\_resource\_group | Resource group name that contains AKS VMs |
+| network\_resource\_group |  Resource group name that contains AKS network resources |
```
Comment (chrismellard) [inline README.md]: Can you keep this list in alpha order please?

**Review by chrismellard — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gregberns) [reply README.md]: @chrismellard done

**Comment by gregberns** [comment]
@chrismellard do you know why the tests keep failing? Not sure where to look to understand why they're failing.

**Comment by chrismellard** [comment]
Yeah, not easy for anyone to currently see failures as the logs are on tin that few have access to. Having run  this locally though via `make test` you can see that terraform is moaning about the Kubernetes version. Looks like Azure have just removed main stream support of k8s 0.15. In another PR I've bumped the cluster_version default to `1.18.8`. So you can either wait for  that PR to merge or make that change here too to get the tests working

**Pull request merged**